app/Notifications/NewRequestBounty.php

```diff
@@ -42,6 +43,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
```

Comment:
Probably shouldn't notify requester if they add bounty to their own request.

Reply:
Added self check.